packages/core/upload/admin/src/components/EditFolderDialog/EditFolderDialog.js

```diff
@@ -42,13 +41,12 @@ export const EditFolderDialog = ({ onClose, folder, parentFolderId }) => {
     enabled: true,
   });
   const { canCreate, isLoading: isLoadingPermissions, canUpdate } = useMediaLibraryPermissions();
-  const submitButtonRef = useRef(null);
   const [showConfirmDialog, setShowConfirmDialog] = useState(false);
   const { formatMessage, formatDate } = useIntl();
-  const { editFolder, isLoading: isEditFolderLoading } = useEditFolder();
+  const { editFolder } = useEditFolder();
   const { remove } = useBulkRemove();
   const toggleNotification = useNotification();
-  const isLoading = isLoadingPermissions || folderStructureIsLoading || isEditFolderLoading;
+  const isLoading = isLoadingPermissions || folderStructureIsLoading;
```

Comment:
Why did you remove the `isEditFolderLoading`? The loading state needs to be displayed, if a request is happening I think.

Reply:
the formik error message wasn't displayed because when you submitted isEditFolderLoading was removing all inputs to show the loading state. 
that's the only solution I found to make the error state/message working, and it doesn't seem to break anything - don't hesitate to test and tell me if it's breaking anything! we can add a loading state to the save button though, we do it in many forms in Strapi, WDYT?

Reply:
Mh, maybe we need split the logic then, because I find it still helpful to see the loading state on e.g. the submit button. Otherwise the user doesn't have an indication of what is going on.

Could you add it back there, if you agree?

Reply:
sorry not sure I understand, do you mean adding a loading state to the submit button?

Reply:
Yes. Not showing the loader seems fine to me, but I'd like to have the state still on the submit button at least.

Reply:
fixed [here](https://github.com/strapi/strapi/pull/13494/commits/a23a176e760cbfd309f91dd3d6a7f9bbc3ecdd3b)